documentation/dev/plugins.rst

```diff
@@ -3,24 +3,35 @@
 Writing Plugins
 ====================
 
-You can extend FlexMeasures with functionality like UI pages or CLI functions.
+You can extend FlexMeasures with functionality like UI pages, API endpoints, or CLI functions.
+This is eventually how energy flexibility services are built on top of FlexMeasures!
 
-A FlexMeasures plugin works as a `Flask Blueprint <https://flask.palletsprojects.com/en/1.1.x/tutorial/views/>`_.
+In an nutshell, a FlexMeasures plugin adds functionality via one or more `Flask Blueprints <https://flask.palletsprojects.com/en/1.1.x/tutorial/views/>`_.
 
 .. todo:: We'll use this to allow for custom forecasting and scheduling algorithms, as well.
 
 
-How it works
-^^^^^^^^^^^^^^
+How to make FlexMeasures load your plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I keep reading that "The underline symbol must extend as long as the text, and appear immediately underneath the name of the section" (which always seemed like a silly rule to me), but this header actually shows up correctly as is. Better safe than sorry I guess. (Also line 136.)

[Some](https://rest-tutorial.readthedocs.io/en/latest/Basic_reStructuredText.html#section-headers) [refs](https://sphinx-tutorial.readthedocs.io/step-1/?highlight=underline#sections)